aspnet/getting-started.rst

```diff
@@ -0,0 +1,53 @@
+Getting Started
+===============
+
+1. Install `.NET Core`_
+
+2. Create a new .NET Core project:
+
+  .. code-block:: console
+    
+    mkdir aspnetcoreapp
+    cd aspnetcoreapp
+    dotnet new
+
+3. Update your ``project.json`` file to add the package for the Kestrel HTTP server and the ASP.NET Core hosting model:
```

Comment:
*project.json* is italic, not code.
I prefer to refer to **THE** file, The project in lieu of YOUR file, YOUR project, etc